stream_alert/rule_processor/handler.py

```diff
@@ -24,6 +27,10 @@
 from stream_alert.rule_processor.rules_engine import StreamRules
 from stream_alert.rule_processor.sink import StreamSink
 from stream_alert.shared.metrics import Metrics
+from stream_alert.shared.stats import time_me
+
+NUM_WORKERS = 6
```

Comment:
Some experimentation might be necessary to know what size works best in Lambda. The standard when creating multiprocessing pools is to default to `multiprocessing.cpu_count()`. I wonder what Lambda would return for `multiprocessing.cpu_count()`?